Adds wxSimpleBook XRC export code.

OmniBlade opened the pull request:
Implements the missing XRC import/export code in plugins/containers/container.cpp.
Fixes #666

sodevel commented on plugins/containers/containers.cpp:
```diff
@@ -892,6 +904,21 @@ class SimplebookPageComponent : public ComponentBase
+    
+    ticpp::Element* ExportToXrc(IObject* obj) override {
+		ObjectToXrcFilter xrc(obj, _("simplebookpage"));
+		xrc.AddProperty(_("label"),_("label"),XRC_TYPE_TEXT);
+		xrc.AddProperty(_("select"),_("selected"),XRC_TYPE_BOOL);
```
Space after comma missing

sodevel commented on plugins/containers/containers.cpp:
```diff
@@ -892,6 +904,21 @@ class SimplebookPageComponent : public ComponentBase
+	ticpp::Element* ImportFromXrc(ticpp::Element* xrcObj) override {
+		XrcToXfbFilter filter(xrcObj, _("simplebookpage"));
+		filter.AddWindowProperties();
+		filter.AddProperty(_("label"),_("label"),XRC_TYPE_TEXT);
+		filter.AddProperty(_("selected"),_("select"),XRC_TYPE_BOOL);
```
Space after comma missing

sodevel reviewed (changes requested):
This is looking good, there are just some minor formatting issues, it would be nice if you could fix them.

OmniBlade commented:
Looks like the spacing issues is prevelent in the file as I just copy pasted those sections and modified them :D I've fixed them now, but left the rest of the file as is so the PR just reflects the functional change.

sodevel commented:
Formatting is kind of a mess in the code base, the long term goal is to implement a format specification to allow automatic formatting of the code. The difficulty is to find a free solution that works on all three platforms easily, so far i only know about clang-format, but i am quite unhappy with some of the results it produces.

Anyway, thanks for the PR, i fixed the introduced whitespace errors myself.
